### ISSUE TITLE
SancoaleSlab font is very hard to read

### ISSUE THREAD

**Issue by duckinator** [rubygems/bundler-site#1074]
Using Firefox on Windows 10, when I go to https://bundler.io/v2.4/man/gemfile.5.html it looks like this:

![image](https://user-images.githubusercontent.com/39698/211682784-c03892b7-dc54-4338-b0a5-d3b95ddfbba5.png)

<br><br>

It's very squished, and sometimes uppercase letters are different heights:

![image](https://user-images.githubusercontent.com/39698/211683052-f23795fa-523b-4535-ad13-d26a68205464.png)

<br><br>

I'm unsure if this is a font problem, a Firefox problem, or a Windows 10 problem.

**Workaround:** Firefox's "Reader view" works well enough for me to use the site.

**Comment by simi** [comment]
Do you have same problem with other browsers? It looks ok on my side (FF on Linux). I can check also FF on Windows later.

![image](https://user-images.githubusercontent.com/193936/212167572-2214965d-2f97-4aa2-8362-1de7ef847de2.png)

**Comment by tnir** [comment]
I totally agree with @duckinator about the readability of that font. I would like to change it with more popular ones like Georgia (see https://ruby-doc.org/3.2.0/).
I have thought it for a longe time before #701 while the font was introduced in #218 and I am totally not sure if the font is properly licensed (purchased or freely licensed).

On my side, `<strong>` does not work well (Google Chrome 108 on macOS 13).
![bundler io_v2 4_man_gemfile 5 html](https://user-images.githubusercontent.com/10229505/212195517-cca9cde4-41f8-4690-a296-dc426b9f5550.png)

**Comment by indirect** [comment]
IIRC we paid for a license to use Sancoale as a webfont. I don't know why @duckinator is seeing such terrible results, but that's definitely a bug in my opinion. I think just changing the body font to something more standard and easier to read would be reasonable. 👍🏻

**Comment by simi** [comment]
If I understand it well, this font is related to `bundler.io` only, not being used at `rubygems.org` typography. I don't see any problem to replace it if needed. :+1:

**Comment by duckinator** [comment]
Same problem in Edge on Windows 10.

![image](https://user-images.githubusercontent.com/39698/213327140-a0243ebd-bf2e-4a99-b400-666080c3472d.png)

Could not reproduce in Firefox on Android.

**Comment by tnir** [comment]
This issue was postponed to [1Q 23](https://github.com/rubygems/bundler-site/milestone/16), but is likely to be postponed to [2Q 23](https://github.com/rubygems/bundler-site/milestone/17).
